portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// async function
+async function displayText() {
+    const serverResponse = await fetch('/hello');
```

Comment:
Where is fetch defined?

Reply:
I used fetch on line 36 as a method. From what I understood, isn't fetch just a method to be called when we want to get the content from server? Do we need to define it?

Reply:
Oops! You're right. My JavaScript knowledge is getting out of date.